ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx

```diff
@@ -115,7 +115,8 @@ const AppointmentEditorCommonFieldsWrapper = props => {
                     <div data-testid="location-search">
                         <LocationSearch value={appointmentDetails.location}
                                         onChange={(optionSelected) => updateAppointmentDetails({location: optionSelected})}
-                                        isDisabled={componentsDisableStatus.location}/>
+                                        isDisabled={componentsDisableStatus.location}
+                                        autoFocus={componentsDisableStatus.speciality && componentsDisableStatus.service && editAutoFocus}/>
```

Comment:
what is editAutoFocus??

Reply:
wheather it need autoFocus or not, which will come from editAppointment.jsx

Reply:
AppointmentEditorCommonFieldsWrapper does not care if it is for edit or not, so why is it called editAutoFocus?

Reply:
it need to know if its edit or add screen. So we are passing editAutoFocus from editAppointment